crates/matrix-sdk-ui/tests/integration/main.rs

```diff
@@ -84,7 +84,7 @@ async fn mock_context(
 }
 
 /// Mocks the /event endpoint
-#[allow(clippy::too_many_arguments)] // clippy you've got such a fixed mindset
+#[allow(clippy::too_many_arguments, dead_code)] // clippy you've got such a fixed mindset
```

Comment:
Why the `dead_code`? If it's unused, let's get rid of it.

Reply:
It's a helper for mocking data, so I expected it to be useful in the future. I can remove it though.